PICS/Onboarding/Welcome.swift

```diff
@@ -16,7 +16,7 @@ struct Welcome: View {
     
     var body: some View {
         OnboardingView(
-            title: "WELCOME_TITLE",
+            title: "APPTQ_TITLE",
```

Comment:
Do we want to keep the original "WELCOME_TITLE"? I found this in the first onboarding page and it looks a bit weird.
![image](https://github.com/CS342/2024-PICS/assets/32094663/ee578d9f-913c-4388-b750-40477e7fbc25)